### PR TITLE
Portfolio: Mobile version

### PR DESCRIPTION
**Pull request by ArnaudBand**
Add a contact form with :
- background Image and background-colour
- Title of my contact form
- Paragraph of my contact form
- A form with input type text and email and textarea and button for submitting

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -173,6 +173,31 @@ <h2>About <br> Myself</h2>
                     <li class="list-item"><a href="#">Skills</a> <i class="fa fa-angle-right"></i></li>
                 </ul>
             </section>
+            <section class="contact_form">
+              <div class="contact-div">
+                <h2 class="contact-h2">Contact me</h2>
+                <p class="contact-paragraph">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+                </p>
+                <form action="/contact-me" method="post">
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider that action value should be similar to `https://formspree.io/f/{form_id}`. Actually, your form is not submitting - just redirect to the not available `contact-me` page.

File: index.html [hunk 2]
```diff
@@ -173,6 +173,31 @@ <h2>About <br> Myself</h2>
...
+                        <li class="form-contact">
+                            <input type="email" name="email" placeholder="Type your email">
+                        </li>
+                        <li class="contact-form">
+                            <textarea name="message" cols="35" rows="10">Type your message</textarea>
+                        </li>
+                        <li class="contact-form">
+                            <button type="submit" class="btn">Get it touch</button>
+                        </li>
+                    </ul>
```
Comment (elmar8287) [inline index.html]: - [ ] You can use simple form structure, without the `ul`/`li`. I know, you can fix it easily 👍

File: index.html [hunk 3]
```diff
@@ -173,6 +173,31 @@ <h2>About <br> Myself</h2>
...
+                <form action="/contact-me" method="post">
+                    <ul class="contact">
+                        <li class="form-contact">
+                            <input type="text" name="name" placeholder="Type Your Name">
+                        </li>
+                        <li class="form-contact">
+                            <input type="email" name="email" placeholder="Type your email">
+                        </li>
+                        <li class="contact-form">
+                            <textarea name="message" cols="35" rows="10">Type your message</textarea>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider adding the `required` for the `textarea`
- [ ] Please, try to remove the text `Type your message` and give `placeholder` 👍 
- [ ] Please, consider that your `teaxtarea` has `maxlength="500"` attribute.

File: index.html [hunk 4]
```diff
@@ -173,6 +173,31 @@ <h2>About <br> Myself</h2>
...
+                <p class="contact-paragraph">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+                </p>
+                <form action="/contact-me" method="post">
+                    <ul class="contact">
+                        <li class="form-contact">
+                            <input type="text" name="name" placeholder="Type Your Name">
+                        </li>
+                        <li class="form-contact">
+                            <input type="email" name="email" placeholder="Type your email">
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider adding the `required` for inputs 👍 
- [ ] Please, consider that the name input has `maxlength="30"` attribute 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- No linter issues 👍 
- Nice design, considering the Figma template! 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- Please, consider creating an informative README file. You can use [MICROVERSE template](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -173,6 +173,21 @@ <h2>About <br> Myself</h2>
                     <li class="list-item"><a href="#">Skills</a> <i class="fa fa-angle-right"></i></li>
                 </ul>
             </section>
+            <section class="contact_form">
+              <div class="contact-div">
+                <h2 class="contact-h2">Contact me</h2>
+                <p class="contact-paragraph">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+                </p>
+                <form action="https://formspree.io/f/form_id" method="post">
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly your form still not working, First you have to create an account on [Formspree service](https://formspree.io/html) after that add a new form like in this pic
image

After that open, the integration tab and you will find your link with the id. Take it cope and replace it with this line `https://formspree.io/f/form_id`.

image

Try to test your form first before you request a review again the page after submitting should say `Thank you` work, not `Form not found.

Your page currently says after submitting 
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @B77748, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @B77748, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**